doc/cookbook/source/examples/evaluation/cross_validation.rst

```diff
@@ -0,0 +1,101 @@
+============================================
+Cross Validation
+============================================
+
+Cross Validation is a type of model validation technique for figuring out how the
+results from a statistical analysis will generalize to an independent data set. It
+essentially involves training the model on a subset of the data, and testing on the
+remainder, then repeating this process for several complementary partitions and then
+combining all the validation results generated (For instance, an average) to estimate
+a performance for the final predictive model.
+
+One type of cross validation is known as :math:`k-` fold cross validation.
+
+For instance, Suppose you have the training data :math:`\mathbf{x}_1,\mathbf{x}_2, ..., \mathbf{x}_{10n}`
```

Comment:
* Remove "For instance"
* replace "you" with "we"
* Please just use "n" data, not 10n

Reply:
Done